tests/test_pfs.py

```diff
@@ -334,6 +334,17 @@ def test_inspect_commit():
     assert len(commit.commit.id) == 32
     assert commit.commit.repo.name == repo_name
 
+def test_squash_commit():
+    client, repo_name = sandbox("squash_commit")
+
+    with client.commit(repo_name, 'master') as c:
```

Comment:
nit

```suggestion
    with client.commit(repo_name, 'master'):
```

I'm planning on enabling the linter for tests as well. Unused variable names would cause the linter to complain.